pkg/auth/authorizer/interfaces.go

```diff
@@ -56,12 +59,20 @@ func (f AuthorizerFunc) Authorize(a Attributes) error {
 	return f(a)
 }
 
+type MetaResource int
+
+const (
+	ApiVersions MetaResource = 1 << iota
+	HealthCheck
+)
+
 // AttributesRecord implements Attributes interface.
 type AttributesRecord struct {
-	User      user.Info
-	ReadOnly  bool
-	Namespace string
-	Resource  string
+	User      		user.Info
+	ReadOnly  		bool
+	Namespace 		string
+	Resource  		string
+	MetaResource 	MetaResource
```

Comment:
@deads2k thoughts on preferred terminology for a field in authorizer.Attributes to convey a non-resource path?